### ISSUE TITLE
Multibody Tree Builder should use full path names for bodies and joints

### ISSUE THREAD

**Issue by chrisdembia** [opensim-org/opensim-core#763]
Right now, the tree builder is fed just the names of bodies and joints instead of their full path names. However, it's possible to have different bodies with the same name in different parts of the model. To facilitate nested models, the tree builder should instead use the unique full path names.

**Comment by jenhicks** [comment]
@aseth1 Could you check if this has been addressed?

**Comment by jenhicks** [comment]
Whevever it says component GetName() use get full path name

**Comment by aseth1** [comment]
Specifically in `Model::createMultibodyTree()` where Bodies and Joints are added to the `_multibodyTree`:
```cpp
...
    for (auto& body : bodies) {
        _multibodyTree.addBody( body.getName(),  // ->getAbsolutePathName()
        ...
...
    for (auto& joint : joints) {
        std::string name = joint->getName();         // ->getAbsolutePathName()
        ...
        // Use joints to define the underlying multibody tree
        _multibodyTree.addJoint(name, ...
```

**Comment by aseth1** [comment]
fixed by #1421
